src/stores/index.js

```diff
@@ -358,16 +417,19 @@ const Store = Reflux.createStore({
    * @param {Connection} connection - The connection to delete.
    */
   onDeleteConnectionClicked(connection) {
-    const toDestrioy = this.state.fetchedConnections.find(
+    const toDestroy = this.state.fetchedConnections.find(
       (item) => item._id === connection._id
     );
 
-    toDestrioy.destroy({
+    toDestroy.destroy({
       success: () => {
```

Comment:
this is Ampersand right :/?

Reply:
Looks like it yup https://ampersandjs.com/docs/#ampersand-model-destroy

Reply:
Connection storage is a big mix of ampersand and storage mixin - maybe we rewrite a storage module once we monorepo?